blocks/library/nextpage/editor.scss

```diff
@@ -0,0 +1,40 @@
+.editor-visual-editor__block[data-type="core/nextpage"] {
+	max-width: 100%;
+}
+
+.wp-block-nextpage {
+	display: block;
+	text-align: center;
+	overflow: hidden;
+	white-space: nowrap;
+}
+
+.wp-block-nextpage > span {
+	position: relative;
+	display: inline-block;
+	font-size: 12px;
+	text-transform: uppercase;
+	font-weight: 600;
+	font-family: $default-font;
+	color: $dark-gray-300;
+}
+
+.wp-block-nextpage > span:before,
+.wp-block-nextpage > span:after {
+	content: '';
+	position: absolute;
+	top: 50%;
+	width: 100vw;
+	border-top: 3px dashed $light-gray-700;
+	margin-top: -2px;
+}
+
+.wp-block-nextpage > span:before {
+	right: 100%;
+	margin-right: 20px;
+}
+
+.wp-block-nextpage > span:after {
+	left: 100%;
+	margin-left: 20px;
+}
```

Comment:
This file is a lot different from `blocks/library/more/editor.scss`. I prefer the final result for Next Page, and we'd probably want to converge a bit on the two stylesheets. Any advice from @jasmussen is much appreciated, as always.